### PR TITLE
Updated docs for smtp4dev connection

### PR DESCRIPTION
**Pull request by BOHEUS**
Taken from [Discord #help ticket](https://discord.com/channels/1130383047699738754/1309967928640147476)

### CODE REVIEW

File: packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx [hunk 1]
```diff
@@ -126,6 +126,7 @@ yarn command:prod cron:calendar:calendar-event-list-fetch
     - Set the following env variables:
       - EMAIL_SMTP_HOST=localhost
       - EMAIL_SMTP_PORT=2525
+      - EMAIL_DRIVER=smtp
```
Comment (martmull) [inline packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx:129]: Nice, can you add it also for google and Office configs? lines 108 an 118

**Review by martmull — COMMENTED** [review]
Nice, thank you. Small additional request then I will approve

File: packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx (hunk 1 above)
Comment (BOHEUS) [reply packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx:129]: If so, wouldn't it be better to simply change it in env.example in twenty-server?

**Review by martmull — APPROVED** [review]
Added it to google and office also, thank you!

File: packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx (hunk 1 above)
Comment (martmull) [reply packages/twenty-website/src/content/developers/self-hosting/self-hosting-var.mdx]: think we should leave "logger" as default value as it does not require smtp configuration

**Pull request merged**